Content.Shared/Clothing/Components/Clown/Waddle.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Numerics;
+
+namespace Content.Shared.Clothing.Components.Clown;
+
+/// <summary>
+/// Defines something as causing waddling when worn.
+/// </summary>
+[RegisterComponent]
+public sealed partial class WaddleComponent : Component
```

Comment:
The component name should better reflect that it goes on entities that make the wearer waddle, rather than directly on entities that should inherently waddle. As it is, a contributor looking to make ducks waddle might be tempted to put WaddleComponent directly on the duck prototype.

WaddleWearer, WaddleWearable, WaddleWhenWorn? Something like that.